plugins/source/aws/resources/services/redshift/clusters_fetch.go

```diff
@@ -20,7 +20,7 @@ func fetchRedshiftClusters(ctx context.Context, meta schema.ClientMeta, parent *
 		if err != nil {
 			return err
 		}
-		res <- response.Clusters
```

Comment:
This change doesn't seem right

Reply:
ah thanks 😮‍💨

Reply:
Reverted 👍

Reply:
Time for me to take a break from the screen I think 😄